client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx

```diff
@@ -35,14 +41,14 @@ const DesksFilterCard = (props: Props): JSX.Element => {
                     />
                     <div className={classes.desksWrapper}>
                         {
-                            desks.map((desk: Desk) => (
+                            countyDesks.map((desk: Desk) => (
                                 <CustomCheckbox
                                     checkboxElements={[{
                                         key: desk.id,
                                         value: desk.id,
-                                        checked: filteredDesks.includes(desk.id),
+                                        checked: filteredDesks.includes(desk.id as number),
                                         labelText: desk.deskName,
-                                        onChange: () => onDeskClicked(desk.id)
+                                        onChange: () => onDeskClicked(desk.id as number)
```

Comment:
The desk.id as number is a reccuring theme here, is it because of the (number | null)[ ] thing? , If so you can guarantee that the value is not null by adding '!' at the end of the row (https://www.logicbig.com/tutorials/misc/typescript/non-null-assertion-operator.html)